two-node-cluster/2-node-cluster.yml

```diff
@@ -650,7 +654,7 @@ Parameters:
   SessionDBAllocationStorage:
     Description: Provide storage size in Gigabytes
     Type: Number
-    Default: 50
+    Default: 100
```

Comment:
Shall we parameterize the db storage as well?

Reply:
I am thinking of adding a high concurrency mode in the next PR. With that we don't need parameters for storage since storage is proportional to IOPS

Reply:
ack